src/modules/epsr/process.cpp

```diff
@@ -293,10 +298,22 @@ Module::ExecutionResult EPSRModule::process(ModuleContext &moduleContext)
         // Calculate r-factor over fit range and store
         auto tempRefData = originalReferenceData;
         Filters::trim(tempRefData, qMin_, qMax_);
-        auto rFactor = Error::rFactor(tempRefData, weightedSQ.total()).error;
+        auto errorReport = Error::rFactor(tempRefData, weightedSQ.total());
+        lowerBound = errorReport.firstX;
+        upperBound = errorReport.lastX;
+        auto rFactor = errorReport.error;
         rFacTot += rFactor;
         errors.addPoint(moduleContext.dissolve().iteration(), rFactor);
```

Comment:
This would then become...
```suggestion
        rFacTot += rFactorReport.error;
        errors.addPoint(moduleContext.dissolve().iteration(), rFactorReport.error);
```